specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/examples/PricesheetDownload.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parameters": {
+    "api-version": "2022-10-01",
+    "billingAccountName": "7c05a543-80ff-571e-9f98-1063b3b53cf2:99ad03ad-2d1b-4889-a452-090ad407d25f_2019-05-31",
+    "billingProfileName": "2USN-TPCD-BG7-TGB",
+    "invoiceName": "T000940677"
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Location": "https://management.azure.com:443/providers/Microsoft.Billing/billingAccounts/7c05a543-80ff-571e-9f98-1063b3b53cf2:99ad03ad-2d1b-4889-a452-090ad407d25f_2019-05-31/billingProfiles/2USN-TPCD-BG7-TGB/providers/Microsoft.CostManagement/operationResults/45000000-0000-0000-0000-000000000000?sessiontoken=0:000000&api-version=2022-10-01&OperationType=PriceSheet",
+        "Retry-After": "60",
+        "OData-EntityId": "45000000-0000-0000-0000-000000000000"
+      }
+    },
+    "200": {
+      "body": {
+        "url": "https://myaccount.blob.core.windows.net/?restype=service&comp=properties&sv=2015-04-05&ss=bf&srt=s&st=2015-04-29T22%3A18%3A26Z&se=2015-04-30T02%3A23%3A26Z&sr=b&sp=rw&spr=https&sig=G%2TEST%4B",
```

Comment:
>sig [](http://example.com/codeflow?start=192&length=3)

FWLIW - this looks like the kind of uri credential scanning might ding you for.

Reply:
added x-ms-secret tag. This is just a dummy url, hopefully doesn't get picked up in scans!